packages/ERTP/src/purse.js

```diff
@@ -28,7 +28,7 @@ export const vivifyPurseKind = (
   //   that created depositFacet as needed. But this approach ensures a constant
   //   identity for the facet and exercises the multi-faceted object style.
   const { depositInternal, withdrawInternal } = purseMethods;
-  const makePurseKit = vivifyFarClassKit(
+  const makePurseKit = defineExoKitFactory(
```

Comment:
I like the idea that a "factory" should be an object with methods for making things, rather than just a function. The second commit tries this out, shifting to the "maker" terminology you suggest below. The second commit also shifts from "define*" to "prepare*". I updated the PR comment to explain the rationale for these new choices.

Reply:
To me, "factory" implies "make a thing", which can happen multiple times as needed; "define" implies "establish a thing", which should only happen once; "prepare" implies "make sure a thing is set up to be the way it is supposed to be", which in principle only should happen once but you might not be in a position to know if it is so you do it as needed just to be sure.  One (English) grammatical issue is that "factory" is a noun identifying something that makes things rather than the operation of making things, which ties to your "methods for making things rather than just a function" remark.